text/0995-deprecate-non-colocated-components.md

```diff
@@ -0,0 +1,247 @@
+---
+stage: accepted
+start-date: 2023-12-15T00:00:00.000Z
+release-date:
+release-versions:
+teams: # delete teams that aren't relevant
+  - cli
+  - framework
+  - learning
+  - typescript
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/995
+project-link:
+---
+
+<!---
+Directions for above:
+
+stage: Leave as is
+start-date: Fill in with today's date, 2032-12-01T00:00:00.000Z
+release-date: Leave as is
+release-versions: Leave as is
+teams: Include only the [team(s)](README.md#relevant-teams) for which this RFC applies
+prs:
+  accepted: Fill this in with the URL for the Proposal RFC PR
+project-link: Leave as is
+-->
+
+# Deprecate non-co-located components.
+
+## Summary
+
+Deprecates
+- classic component layout
+- pods component layout
+
+
+After "ember@6", only the following will be allowed:
```

Comment:
I feel like the rfc shouldn't specify what version it's trying to target. It is true that we're trying to get it for ember@6 but if we don't get this RFC implemented in the next 12 weeks then we miss that boat, so it shouldn't be codified in the RFC but something that we're aiming for anyway 👍

Reply:
this is a fair point, I will update